src/frontend/src/pages/Index/Settings/AdminCenter/UserManagementPanel.tsx

```diff
@@ -26,22 +26,22 @@ export default function UserManagementPanel() {
           <GroupTable />
         </Accordion.Panel>
       </Accordion.Item>
-      <Accordion.Item value='settings' key='settings'>
+      <Accordion.Item value='tokens' key='tokens'>
         <Accordion.Control>
-          <StylishText size='lg'>{t`Settings`}</StylishText>
+          <StylishText size='lg'>{t`Access Tokens`}</StylishText>
```

Comment:
```suggestion
          <StylishText size='lg'>{t`Tokens`}</StylishText>
```
Access Tokens seems redundant